add an embedding bag skeleton operators

hyuen opened the pull request:
Summary:
basic reference embedding bag operators, these are not going to be performant but can be used for functionality enablement

these operators will output the right shape, but the implementation is empty

Test Plan: tbd

Differential Revision: D29313058

ezyang commented:
pro tip: you can use ghexport to preserve the stack in GH without repeating the changes here

Commit:
add an embedding bag skeleton operators (#60491)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/60491

basic reference embedding bag operators, these are not going to be performant but can be used for functionality enablement

these operators will output the right shape, but the implementation is empty

Test Plan: tbd

Reviewed By: vkuzo

Differential Revision: D29313058

fbshipit-source-id: 134ee75da7539c51dfc61b5a398f77611c19bc4c
